sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/BlobChangeFeedClientBuilderExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.Extensions;
+using Azure.Storage;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.ChangeFeed;
+
+namespace Microsoft.Extensions.Azure
```

Comment:
This is not the correct namespace where our SDK lives. If it's relevant to changefeed it has to live in `Azure.Storage.Blobs.ChangeFeed` as for extensions I believe there's already an existing class for `BlobChangeFeedExtensions`

Reply:
Oops, sorry!
I was following the pattern in [BlobClientBuilderExtensions](https://github.com/Azure/azure-sdk-for-net/blob/59821ed753e23cc70c1edfa1d08a3d37305d71df/sdk/storage/Azure.Storage.Blobs/src/BlobClientBuilderExtensions.cs).

But having a closer look in `BlobChangeFeedExtensions` I think I've found what I need anyway in `GetChangeFeedClient()` :)

Should I close this PR?

Reply:
@amnguye this is a correct namespace for these extensions methods.https://azure.github.io/azure-sdk/dotnet_implementation.html#dotnet-client-builder-class-namespace

Reply:
Yeah I was just about to say my bad. Yeah I see that the client factories are supposed to be in that extension package

Reply:
All good, thanks for the review. This helped me find what I need :)